packages/insomnia-app/app/ui/components/request-pane.js

```diff
@@ -82,6 +92,17 @@ class RequestPane extends React.PureComponent<Props> {
     });
   }
 
+  async _handlePluginTabs() {
+    getRequestTabs().then(
+      pluginTabs => this.setState({ pluginTabs }),
+      err => console.error(err),
+    );
+  }
+
+  componentWillMount() {
+    this._handlePluginTabs();
+  }
```

Comment:
I think we should do this in the `componentDidMount` lifecycle event, otherwise the request-pane will be slower to load initially if there are multiple plugins. Follows a similar pattern to [config generator plugins](https://github.com/Kong/insomnia/blob/3314c23ecfb95cd3b9357621a55daa4de76242e9/packages/insomnia-app/app/ui/components/wrapper-design.js#L142-L147).

 ```suggestion
  async componentDidMount() {
    await this._handlePluginTabs();
  }
```

Reply:
Fixed in d53beaf8